src/StoreKeeper/WooCommerce/B2C/Core.php

```diff
@@ -145,6 +145,10 @@ public static function plugin_abspath()
      */
     public function __construct()
     {
+        add_action('init', function() {
+            load_plugin_textdomain('free-shipping', false, 'storekeeper-for-woocommerce/languages/');
+        });
```

Comment:
Use translation files in `i18n` , those files are the `I18N::DOMAIN` . Change domain for all translations in here to `I18N::DOMAIN` and extract the translations to them.